.env

```diff
@@ -0,0 +1,7 @@
+PORT=3000
+
+CHAIN_ID=1337
+CHAIN_HOST=http://localhost:7545
+CHAIN_NAME=Localchain
+
+BUILD_ENV=testing
```

Comment:
This sounds a lot like `test` which I don't think is what you mean? I would have expected `local` or `development`

Reply:
Ah now I see that this is torus specific I think? Everything that is just for torus should be prefixed.

Reply:
Yeah the build environment in this case was Torus specific, I'll make that more clear.